internal/session/repository_connection.go

```diff
@@ -184,6 +184,71 @@ func (r *ConnectionRepository) ListConnectionsBySessionId(ctx context.Context, s
 	return connections, nil
 }
 
+// CloseConnectionResp is just a wrapper for the response from CloseConnections.
+// It wraps the connection and its states for each connection closed.
+type CloseConnectionResp struct {
+	Connection       *Connection
+	ConnectionStates []*ConnectionState
+}
```

Comment:
likewise this can be unexported, since it is a result that only the domain service uses